README.md

```diff
@@ -68,7 +68,7 @@ $ sudo usermod -a -G plugdev USERNAME
 ```
 
 #### Add udev rules
-Copy these two files into your /etc/udev.rules directory:
+Copy these two files into your /etc/udev/rules.d directory:
```

Comment:
How about change like this then:

... /etc/udev/rules.d directory (/etc/udev.rules for Ubuntu 14.04):